app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,7 @@
         android:allowBackup="true"
         android:dataExtractionRules="@xml/data_extraction_rules"
         android:fullBackupContent="@xml/backup_rules"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/my_digital_closet_logo"
```

Comment:
Should be resized to fit the dimensions of the app icon- low priority, I just really like that the icon shows up now truth be told.

Reply:
Yep totally agree. I will focus on it once I finish the other parts of the app.

Reply:
The icon showing is definitely an accomplishment!